### ISSUE TITLE
Our GitHub Action is testing github.com/tjgurwara99/Go/... not TheAlgorithms

### ISSUE THREAD

**Issue by cclauss** [TheAlgorithms/Go#271]
**Describe the bug**
@tjgurwara99 
Our GitHub Action is testing github.com/tjgurwara99/Go/... not github.com/TheAlgorithms/Go/...

**To Reproduce**
1. Look at the path in the output of our GitHub Actions go tests.  We are testing `tjgurwara99`, instead of `TheAlgorithms`.

**Expected behavior**
We test `github.com/TheAlgorithms/Go/...`

**Screenshots**
```
Run go test $(go list ./... | grep -v /sbom | grep -v /rsa)
?   	github.com/tjgurwara99/Go/ciphers	[no test files]
ok  	github.com/tjgurwara99/Go/ciphers/caesar	0.016s
ok  	github.com/tjgurwara99/Go/ciphers/polybius	0.010s
ok  	github.com/tjgurwara99/Go/ciphers/rot13	0.005s
ok  	github.com/tjgurwara99/Go/ciphers/xor	0.006s
```

**Additional context**

**Comment by tjgurwara99** [comment]
That's because of the `go.mod` - in hind sight I shouldn't have committed the file. The file should be updated to work with TheAlgorithm handle.

**Comment by cclauss** [comment]
Can you please submit a PR to fix that and also upgrade go v1.14 --> v1.15 in `go.mod` ?

**Comment by tjgurwara99** [comment]
Sure, will do that now.

**Comment by cclauss** [comment]
Closed in #273
